README.rst

```diff
@@ -148,6 +148,20 @@ The recipe supports the following options:
 **flake8**
     If set to True, run Flake8 code analysis. Default is ``True``.
 
+    Flake8 now takes advantage of ``flake8`` plugin system to cover some of
+    previously manually written linters. We've plugged in following extensions:
+
```

Comment:
(opinion ahead) This sentence makes sense now, but the day after this is released the previous linters will look odd, so maybe rephrasing it in a way that "flake8 is super-powered with the following extensions" maybe would be better